qcengine/programs/torchani.py

```diff
@@ -76,6 +76,9 @@ def ensemble_forward(self, species_input):
         elif name == "ani1ccx":
             self._CACHE[name] = torchani.models.ANI1ccx()
 
+        elif name == "ani2x":
+            self._CACHE[name] = torchani.models.ANI2x()
+
```

Comment:
To handle the v1/v2 compatibility, how about something like:

```
ani_models = {
     "ani1x": torchani.models.ANI1x(),
     "ani1ccx": = torchani.models.ANI1ccx(),
}
if parse_version(self.get_version()) >= parse_version("2.0"):
    ani_models["ani2x"] = torchani.models.ANI2x()

try:
    self._CACHE[name] = ani_models[name]
except KeyError:
    raise InputError(f"TorchANI only accepts methods: {ani_models.keys()}")
```

Reply:
Thanks for the suggestion, I fixed it in 10863d0

Reply:
I am not sure why lint is complaining though!